src/udev/udev-ctrl.c

```diff
@@ -79,14 +78,6 @@ int udev_ctrl_new_from_fd(struct udev_ctrl **ret, int fd) {
                 .bound = fd >= 0,
         };
 
-        /*
-         * FIXME: remove it as soon as we can depend on this:
-         *   http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=90c6bd34f884cd9cee21f1d152baf6c18bcac949
-         */
```

Comment:
We depend on linux >= 3.13, and this commit is v3.12-rc3-337-g90c6bd34f8, so indeed, we can drop this.